pages/events.tsx

```diff
@@ -0,0 +1,231 @@
+    timestamp: "2023-01-25T13:00:00-05:00",
+    location: "online",
+    description: "",
+    link: "https://lu.ma/rp3submissions",
+  },
+  {
+    type: "Workshop",
+    title: "Ready Player 3 Hackathon | Closing Ceremony + Winners Announcement",
+    timestamp: "2023-02-06T13:00:00-05:00",
+    location: "online",
+    description: "",
+    link: "https://lu.ma/rp3closing",
+  },
+];
+
+const EventsPage: ThirdwebNextPage = () => {
+  const [sortBy, setSortBy] = useState<"date" | "location" | "type" | "title">(
+    "date",
+  );
+  const [sortedEvents, setSortedEvents] = useState(allEvents);
+
+  const handleSort = (value: "date" | "location" | "type" | "title") => {
+    setSortBy(value);
+    switch (value) {
+      case "date":
+        setSortedEvents(
+          allEvents.sort((a, b) => {
+            const dateA = new Date(a.timestamp);
+            const dateB = new Date(b.timestamp);
+            return dateA.getTime() - dateB.getTime();
+          }),
+        );
+        break;
+      default:
+        setSortedEvents(
+          allEvents.sort((a, b) => {
+            return a[value] < b[value] ? -1 : 1;
+          }),
+        );
+        break;
+    }
+  };
+
+  return (
+    <DarkMode>
+      <NextSeo title="events" />
+      <Flex
+        sx={{
+          // overwrite the theme colors because the home page is *always* in "dark mode"
+          "--chakra-colors-heading": "#F2F2F7",
+          "--chakra-colors-paragraph": "#AEAEB2",
+          "--chakra-colors-borderColor": "rgba(255,255,255,0.1)",
+        }}
+        justify="center"
+        flexDir="column"
+        as="main"
+        bg="#000"
+      >
+        <HomepageTopNav />
+
+        <Box maxW="100vw" mt="-100px" overflowX="hidden" minH="100vh" pt={20}>
+          <HomepageSection id="header">
+            <Aurora
+              pos={{ left: "50%", top: "50%" }}
+              size={{ width: "2000px", height: "2000px" }}
+              color={"hsl(280deg 78% 30% / 30%)"}
+            />
+
+            <Flex
+              flexDir="column"
+              align="center"
+              gap={12}
+              mt={{ base: 12, md: 24 }}
+            >
+              <Heading size="title.xl" textAlign="center">
+                Explore our Upcoming Events!
+              </Heading>
+            </Flex>
+            <Grid
+              column={{ base: 1, md: featuredEvents.length === 2 ? 2 : 1 }}
+              gap={12}
+              mt={12}
+              placeContent="center"
+            >
+              {featuredEvents.map(({ type, title, date, banner, link }) => (
+                <FeaturedCard
+                  key={title}
+                  type={type}
+                  title={title}
+                  date={date}
+                  banner={banner}
+                  link={link}
+                />
+              ))}
+            </Grid>
+          </HomepageSection>
+
+          <HomepageSection mt={20}>
+            <Select
```

Comment:
Try moving it to the right and reducing the bottom margin because it's equidistant from both the featured cards and events list. If moving it to the right doesn't look good then keep it in place.